openhtf/phasemanager.py

```diff
@@ -139,7 +139,8 @@ def __init__(self, cell_config, test, test_state, capabilities):
     self._config = cell_config
     self._phases = list(test.phases)
     self._test_state = test_state
-    self._logger = test_state.logger
```

Comment:
This is a change we should discuss in person - by not using the test_state.logger, you're changing the behavior so that logs here don't get output to the test record anymore.  We should do an audit and decide which log messages make sense to go to the test record and which should just be console output.  Or perhaps we have a flag to set the log level that gets output to the test record?